README.md

```diff
@@ -3,6 +3,12 @@
 
 A crawler to find websites that exercise code in Firefox that is not covered by unit tests
 
+## Software requirements
+
+- You will need python>=3.5, pip for python>=3.5 and [Mercurial](https://www.mercurial-scm.org/) source control management tools installed on your machine to run the script.
```

Comment:
Hi! You need to use Python 3.6+.

Reply:
Thank you for review! Will update that asap :)

Reply:
Updated